module/VuFind/src/VuFind/View/Helper/Root/RecordDataFormatter.php

```diff
@@ -126,6 +143,10 @@ protected function allowValue($value, $options, $ignoreCombineAlt = false)
      */
     protected function render($field, $data, $options)
     {
+        if (!($options['enabled'] ?? true)) {
+            return null;
```

Comment:
This is a pre-existing error, but return type above should be `?array`.

Reply:
fixed